apps/users/models.py

```diff
@@ -55,6 +59,8 @@ class Student(Base):
     speciality_id = Column(INTEGER, ForeignKey("speciality.speciality_id"), nullable=False)
     user_id = Column(INTEGER, ForeignKey("user.user_id"))
     faculty_id = Column(INTEGER, ForeignKey("faculty.faculty_id"), nullable=False)
+    created_at = Column(DATETIME(timezone=True), default=datetime.utcnow())
+    updated_at = Column(DATETIME(timezone=True), default=datetime.utcnow())
```

Comment:
please nore that in the scope of this PR you didn't change column 'OneTimeToken.expires'

Reply:
And I'm not!

Reply:
@victor-nuzhniy please do it, bcs Kostya forgot to remark about it in ticket

Reply:
ok

Reply:
please fix issues with this field. now endpoints for registration and check student existence don't work correctly

Reply:
Fixed it